packages/graphene.rb

```diff
@@ -3,39 +3,28 @@
 class Graphene < Package
   description 'A thin layer of graphic data types'
   homepage 'https://github.com/ebassi/graphene'
-  version '1.6.0'
-  source_url 'https://github.com/ebassi/graphene/archive/1.6.0.tar.gz'
-  source_sha256 '98970f859e452ce421b72726ca727fdf3ac27cb4804b62bfe520157fa46aa2fd'
+  version '1.8.0'
+  source_url 'https://github.com/ebassi/graphene/archive/1.8.0.tar.gz'
+  source_sha256 '410f2e848952cc5830f39b6f6ea7f9b0a487cfe99dad86eec6f22ccbb3ec635b'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/graphene-1.6.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/graphene-1.6.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/graphene-1.6.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/graphene-1.6.0-chromeos-x86_64.tar.xz',
+
   })
   binary_sha256 ({
-    aarch64: 'cf86736cfe01b752bc93366c84f92b135196d32cb5918b5e73c3b270fb7a4421',
-     armv7l: 'cf86736cfe01b752bc93366c84f92b135196d32cb5918b5e73c3b270fb7a4421',
-       i686: '40545e0b6b19f7f9bff26ca933f1bbfa601b3403bce2dfdec82af6a885b534a3',
-     x86_64: 'efa9a4ea013fbd15604a86c2a858e6918327287390d467481ccc53df2276acd8',
+
   })
 
   depends_on 'meson'
+  depends_on 'gobject_introspection' => :build
 
   def self.build
-    system "meson _build --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} -Denable-arm-neon=false"
+    system "meson --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} -Darm_neon=false _build"           
+    system "ninja -v -C _build"
+    system "ninja -C _build test"      
```

Comment:
This should probably still be included in the `def self.check` section but it's more semantic than incorrect.

Reply:
If i put it there, it is against meson build procedure.

Reply:
No worries, I already merged.